test/built-ins/TypedArray/prototype/fill/BigInt/fill-values-conversion-operations.js

```diff
@@ -1,56 +0,0 @@
-// Copyright (C) 2016 the V8 project authors. All rights reserved.
```

Comment:
I'm removing these bigint conversion because they are not doing anything for BigInts anyway. We need to work on these separately in the future.